tests/foreman/destructive/test_capsule_loadbalancer.py

```diff
@@ -227,3 +227,49 @@ def test_loadbalancer_register_client_using_ak_to_ha_proxy(loadbalancer_setup, r
         {'organization-id': loadbalancer_setup['module_org'].id}
     )
     assert rhel7_contenthost.hostname in [host['name'] for host in hosts]
+
+
+@pytest.mark.tier1
+def test_client_register_through_lb(
+    loadbalancer_setup, rhel7_contenthost, module_target_sat, module_org, module_ak, module_location
+):
+    """Register the client through loadbalancer
+
+    :id: c7e47d61-167b-4fc2-8d1a-d9a64350fdc4
+
+    :Steps:
+
+        1. Try package installation
+        2. run `subscription-manager register --org=Your_Organization \
+            --activationkey=Your_Activation_Key \
+            --serverurl=https://loadbalancer.example.com:8443/rhsm \
+            --baseurl=https://loadbalancer.example.com/pulp/content`
+        3. Register host through loadbalancer using global registration
+
+    :expectedresults: Global Registration should have option to register through
+    loadbalancer and host should get registered successfully.
+
+    :CaseLevel: Integration
+    """
+    url = f'https://{loadbalancer_setup["setup_haproxy"]["haproxy"].hostname}'
+    server_url = f'{url}:8443/rhsm'
+    base_url = f'{url}/pulp/content'
+
+    result = rhel7_contenthost.download_install_rpm(
+        repo_url=f'{url}/pub', package_name='katello-ca-consumer-latest.noarch'
+    )
+    assert result.status == 0
+
+    command = module_target_sat.api.RegistrationCommand(
+        org=loadbalancer_setup['module_org'].label,
+        activation_key=loadbalancer_setup['content_for_client']['client_ak'].name,
+        serverurl=server_url,
+        baseurl=base_url,
+    ).create()
+    result = rhel7_contenthost.execute(command)
+    assert result.status == 0, f'Failed to register host: {result.stderr}'
```

Comment:
can we add some more assertations here for validating subscription_config like hostname or ports?